README.md

```diff
@@ -308,6 +308,9 @@ See [CONTRIBUTING.md](https://github.com/pantheon-systems/pantheon-advanced-page
 
 ## Changelog ##
 
+## Latest ##
+* Adds support for WordPress Multisite which resolves issue where editing a Post on one subsite clears the home page cache of other sites in the Multisite install if it has a Post containing the same ID. For users who already have this plugin installed on a WPMS, they will need to click the Clear Cache button on the settings page to generate the prefixed keys [[#228](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/228)].
```

Comment:
The note in the changelog for users (referenced [here](https://github.com/pantheon-systems/pantheon-advanced-page-cache/pull/228#discussion_r1166078296)) should be moved to the `== Upgrade Notice ==` section of the readme so it is more visible to users than just being in the changelog.